Engine/DataFeeds/FileSystemDataFeed.cs

```diff
@@ -337,7 +337,10 @@ private Subscription CreateUniverseSubscription(SubscriptionRequest request)
             var config = request.Configuration;
 
             // define our data enumerator
-            var enumerator = GetEnumeratorFactory(request).CreateEnumerator(request, _dataProvider);
+            var factory = GetEnumeratorFactory(request);
+            if (factory == null) return null;
```

Comment:
maybe a comment here about why we may get a null enumerator for a subscription request -- if I'm understanding this correctly, it's essentially saying that we're *not* going to act on this subscription request.